### ISSUE TITLE
Trading bot for Phamous

### ISSUE THREAD

**Issue by dhiren1987** [Protoxs/GMX-Trading-Bot-Opensourced#1]
Hi, I wasn't sure how to message you. I was wondering if this bot could be coded to work with Phamous on Pulsechain, a GMX fork.
I would be happy to help test it if this was something you could do.

Thanks

**Comment by Protoxs** [comment]
Hey man, this bot is currently very well optimized for Arbitrum chain GMX, I will check out Pulsechain but if you want to make steady profit I recommend using this bot on the chain I tested it in for 6 months

**Comment by dhiren1987** [comment]
Yeah, I don't think it will work on pulsechain, as currently there a time lock for each trade and not much liquidity to place the trade every time.

There is a massive arbitrage opportunity with these new chains, but i dont have a bot to do it on chain, i did work with a coder to make one for traditional currencies but not crypto.

________________________________
